infra/base/terraform/eks.tf

```diff
@@ -272,7 +270,7 @@ module "eks" {
       # }
       # capacity_reservation_specification = {
       #   capacity_reservation_target = {
-      #     capacity_reservation_id = "cr-abcedefgh" # Replace with your capacity reservation ID
+      #     capacity_reservation_id = var.capacity_block_reservation_id # Replace with your capacity reservation ID
```

Comment:
You can uncomment this capacity block now since this node group is created only when the CBR is defined in variables